.github/workflows/CI.yml

```
@@ -102,6 +102,13 @@ jobs:
       - name: "Run accessibility tests with playwright 🎭"
         # build PST, build docs, then run a11y-tests
         run: python -Im tox run -m a11y
+        continue-on-error: true
+      - name: "Upload Playwright results (including traces, if any) 🐾"
+        uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
+        with:
+          name: playwright-results
+          path: playwright-results/
```

with:
        name: playwright-traces
        path: test-results/
```suggestion
        with:
          name: playwright-results
          path: playwright-results/
```